packages/react-error-overlay/src/utils/parseCompileError.js

```diff
@@ -7,7 +7,7 @@ export type ErrorLocation = {|
   colNumber?: number,
 |};
 
-const filePathRegex = /^\.(\/[^/\n ]+)+\.[^/\n ]+$/;
+const filePathRegex = /^(?:\.|[A-Z]:)?(\/[^/\n ]+)+\.[^/\n ]+$/;
```

Comment:
This regex should match both relative and absolute paths on all platforms.
`(?:\.|[A-Z]:)?` defines an optional non-capturing group which may contain either a period (for matching relative paths) or a Windows drive letter for matching absolute paths. I verified that [Windows does not support drive letters after Z](https://social.technet.microsoft.com/Forums/en-US/f7079c92-b263-479a-9732-2f0fa4fb862c/drive-letter-after-z?forum=w8itprogeneral).
Unix absolute paths (`/path/to/folder`) are also matched correctly as the first group is optional.